Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN yarn && yarn task build:server:binary
 # We deploy with ubuntu so that devs have a familiar environemnt.
 FROM ubuntu:18.10
 WORKDIR /root/project
-COPY --from=0 /src/packages/server/cli-linux /usr/local/bin/code-server
+COPY --from=0 /src/packages/server/cli-linux-x64   /usr/local/bin/code-server
```

Comment:
afaik, there shouldn't be a `-x64` suffix?

@kylecarbs

Reply:
There should be - that is actually the only output

Reply:
Yup. This was recently changed without me knowing. It's fixed in #109.